controllers/article.js

```diff
@@ -13,23 +13,47 @@ class Article {
    * @returns {Object} Article
    */
   static async createArticle(req, res) {
+    const {
+      title, body, taglist, authorid
```

Comment:
I don't think you should be getting author id from the body?

Reply:
Yes, this was changed in later PR, as part of continuous integration.